wiki_guidelines.md

```diff
@@ -0,0 +1,66 @@
+
+## Wiki Structure 
+We have a tree structure for our wiki, with relevant pages bundled together under the same cateogry. 
+* In the Home page you will find a list of contributors to the project. 
+* In the home section, you will find our code of conduct while working within the team as well as the overall timeline of the project. 
+* In the team section there will be a list of meeting minutes as well as the contributions each team member has made. 
+* A best practices section describes our strategy for the git workflow, issue writing, pull request guidelines, commenting guidelines and other best practices developers contributing to the project should follow. 
+* The tech section describes the technology stack used in the project and the reasoning behind our approach. As well other implementation details pertaining to the project. 
+* A single future works page will contain work that has not been started but could be something that future contributors could work on.
+
+
+### Meeting minutes format
+Meeting minutes format should be standardised and should include:
+* Date
+* Agenda
+* Members present 
+* Summary of meeting
+
+### Documenting contributions 
+When documenting contributions to the project, each contributor should be fairly credited with the work they have accomplished. So the contributions page should include a short description of the different work each team member contributed (both technical or non-technical). The contributions page should be split up into the different parts of the project for example. 
+
+```
+DevOps 
+- Casey: I worked on x feature, and y feature and helped on z feature.
+- Reshad: I worked on m feature and helped with x feature 
+... 
+
+Frontend 
+- Jason: I helped build s feature and also worked on p feature
+- Lucy: I worked on p feature and w feature 
+...
+
+Backend
+...
+
+Documentation
+...
+
+And so on..
+```
+
+## Future work
+If you are suggesting future work that someone could work on, it is important that the description of the work is clear what the work entails. Additionally, details on why the feature should be worked on and how to approach it should be given. For example:
+
+```
+ Feature description: I want to add x feature to the front end 
+ Because: I think this feature will help improve y feature
+ Feasibility: I think this is possible if we do y and z to the backend 
+``` 
+
+ ## Code of conduct when contributing 
+ Before you contribute to the wiki it is important you have reviewed our code of conduct. All contributors are expected to contribute while treating their fellow contributors with respect. 
+
+ ## Writing style 
+ Some important writing guidelines when contributing include:
+ * Writing in first person e.g. "we implemented y using ... "
```

Comment:
Is this really necessary?

Reply:
@baja156 Are you referring to why we have a writing style? 

Imo it is important to have a writing style so the way our contributions are written are consistent.

Reply:
About first person, 3rd person is generally considered more formal

Reply:
> About first person, 3rd person is generally considered more formal

I think using first person pronouns like "we" should be fine, as it'd include the group as a whole. I think we'd want to avoid using singular pronouns like "I" or "me".

Reply:
Sounds like a great time 👍 @cwon880 maybe specify this

Reply:
sounds good i will amend this to the guidelines